layouts/partials/opengraph.html

```diff
@@ -1,25 +1,25 @@
-{{ $title := .Params.ogTitle | default .Title | default site.Title }}
+{{- $title := .Params.ogTitle | default .Title | default site.Title -}}
 
-{{ $description := .Params.ogDescription | default .Description | default (.Summary | plainify | htmlUnescape | replaceRE "\\s+" " ") | default site.Params.description | truncate 150 }}
+{{- $description := .Params.ogDescription | default .Description | default (.Summary | plainify | htmlUnescape | replaceRE "\\s+" " " | replaceRE "^ +| +$" "") | default site.Params.description | truncate 150 -}}
 
-<!-- Handle base URL for local dev vs production -->
-{{ $baseURL := .Site.BaseURL | strings.TrimSuffix "/" }}
-{{ if not (strings.Contains $baseURL "localhost") }}
-  {{ if not (strings.HasPrefix $baseURL "https://") }}
-    {{ $baseURL = replace $baseURL "http://" "https://" }}
-  {{ end }}
-{{ end }}
+{{- /* Handle base URL for local dev vs production */ -}}
+{{- $baseURL := .Site.BaseURL | strings.TrimSuffix "/" }}
+{{- if not (strings.Contains $baseURL "localhost") }}
+    {{- if not (strings.HasPrefix $baseURL "https://") }}
```

Comment:
`opengraph.html` was using 2 spaces for indentation, now it's been changed to 4 for consistency.